src/index.html

```diff
@@ -5,9 +5,54 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main class="card" data-qa="card">
+      <div class="card__img">
+        <img
+          class="img"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          >
+      </div>
+      <div class="card__description">
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__description-code">
+          Product code: 195434
+        </p>
+      </div>
+      <div class="card__reviews">
+        <div class="stars stars--4">
```

Comment:
this should block should be using `<ul>` list with `<li>` tags for better semantics

```suggestion
        <ul class="stars stars--4">
          <li class="stars__star"></li>
          <li class="stars__star"></li>
          <li class="stars__star"></li>
```